### PR TITLE
Moved some ansible work into cloudinit and installed docker on all hosts

### PR DESCRIPTION
**Pull request by greg-szabo**
This change doesn't introduce new features. Its main purpose is to introduce a different (possible quicker) way of running commands that we only run onces on the hosts.

- Commands from ansible's base.yml and prometheus-node-exporter.yaml now run during the first boot of the node in DigitalOcean.
- Each node will auto-install docker at first boot.

We don't necessarily need docker. If you think we shouldn't pre-install it, I can take it out from this change.

### CODE REVIEW

**Review by melekes — APPROVED** [review]
👍

**Review by hvanz — APPROVED** [review]
Thanks, @greg-szabo!

**Review by mzabaluev — APPROVED** [review]
:heart_eyes:

File: ansible/base.yaml [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-  vars:
-    ansible_host_key_checking: false
-
-  tasks:
-    - name: Update apt cache
-      ansible.builtin.apt:
-        update_cache: yes
-        cache_valid_time: 60
-      become: yes
-    - name: restart systemd journal
```
Comment (jmalicevic) [inline ansible/base.yaml:14]: Is this present in the new config ? I was not able to get logs from the comet node. If they are not retrievable via journalct anymore, how can we access them?

Comment (hvanz) [reply ansible/base.yaml:14]: Retrieving logs via journalctl was in in #54

**Pull request merged**